diff --git a/ds.ds_fields_info.inc b/ds.ds_fields_info.inc
index 4f824f9..b0a2cf0 100644
--- a/ds.ds_fields_info.inc
+++ b/ds.ds_fields_info.inc
@@ -57,6 +57,9 @@ function ds_ds_fields_info($entity_type) {
     $fields['node']['comments'] = array(
       'title' => t('Comments'),
       'field_type' => DS_FIELD_TYPE_IGNORE,
+      'ui_limit' => array(
+        '*|full', '*|default',
+      ),
     );
   }
 
